deploy/helm/sumologic/conf/events/fluentd/events.conf

```diff
@@ -13,7 +13,7 @@
 {{- else }}
 <source>
   @type events
-  deploy_namespace {{ .Release.Namespace }}
+  deploy_namespace {{ template "sumologic.namespace" . }}
```

Comment:
What about the reference to `{{ $.Release.Namespace }}` in line 5 above?

Reply:
@astencel-sumo Thanks for pointing that out. I have replaced all remaining occurrences of .Release.Namespace